pkg/util/ovs/ovs.go

```diff
@@ -244,6 +248,22 @@ func (ovsif *ovsExec) Set(table, record string, values ...string) error {
 	return err
 }
 
+// Returns any UUIDs of records that match the condition
+func (ovsif *ovsExec) Find(table, condition string) ([]string, error) {
+	output, err := ovsif.exec(OVS_VSCTL, "find", table, condition)
+	if err != nil {
+		return nil, err
+	}
+	uuids := make([]string, 0)
+	for _, line := range strings.Split(output, "\n") {
+		parts := strings.Split(line, ":")
+		if len(parts) >= 2 && strings.TrimSpace(parts[0]) == "_uuid" {
```

Comment:
You can force ovs-vsctl to do all this for you. `--no-heading --columns=_uuid find ...`